### PR TITLE
Apim 635 docker image for load tests

### PR DESCRIPTION
**Pull request by leleueri**
## Issue

https://gravitee.atlassian.net/browse/APIM-635

## Description

Provide a docker image to run k6 simulations.
It will be useful to execute simulation from kubernetes cluster for example and/or to automate load tests

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-icuerrmndg.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/apim-635-docker-image-for-load-tests/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-perf/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+FROM grafana/k6:0.42.0@sha256:aba50200f5f832925bfc20d2fd822f522f83a6ee794233d46f8837588eba5799 as runner
+
+USER root
+RUN apk add --update --no-cache jq util-linux bash
+USER k6
+
+RUN mkdir -p /tmp/perf_report/k6 \
+    && chown -R 12345:12345 /tmp/perf_report/k6
```
Comment (ytvnr) [inline gravitee-apim-perf/Dockerfile:21]: not sure to understand the reason behind `12345`

**Review by ytvnr — APPROVED** [review]
💪

File: gravitee-apim-perf/Dockerfile (hunk 1 above)
Comment (leleueri) [reply gravitee-apim-perf/Dockerfile:21]: this is the id of the k6 user and the k6 group provided by the k6 image. 
this is to be sure that k6 CLI will be able to write inside this dir

File: gravitee-apim-perf/README.md [hunk 2]
```diff
@@ -178,3 +178,21 @@ Notice that client methods from [src/lib/clients](src/lib/clients) uses a specia
...
 ---
 
+## Docker 
+
+A docker image that embeds all the simulations can be created.
+```bash
+$ docker build -t apim-k6-runner .
+```
+
+Once the image is available, you can run a simulation by providing the path as parameter (relative to the scr/scenario directory).
```
Comment (jgiovaresco) [inline gravitee-apim-perf/README.md]:
```suggestion
Once the image is available, you can run a simulation by providing the path as parameter (relative to the src/scenario directory).
```

**Review by jgiovaresco — APPROVED** [review]

**Pull request merged**